db/migrate/20220121115116_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Following the project requirements. It will be nice to use `:author` as the foreign_key here instead of `:user`.

Reply:
This issue is well resolved! ✔️

Reply:
This issue is well resolved! ✔️